lib/minimagick/tool.rb

```diff
@@ -0,0 +1,16 @@
+# Hack to monkey-patch MiniMagick to always add the 'quiet'
+# option to every imagemagick command line.
+class MiniMagick::Tool
+  class_attribute :quiet_arg
+  self.quiet_arg = false
+
+  prepend(Module.new do
+    def command
+      if quiet_arg
+        [*executable, *(['-quiet'] + args)]
+      else
+        super
+      end
+    end
+  end)
+end
```

Comment:
You mean you think that the location I put it in will possibly cause it to be loaded when we didn't intend? that makes sense. I like `lib/minimagick_patch`.